en/getting-started/first-game/index.md

```diff
@@ -362,165 +370,190 @@ export class GameManager extends Component {
+    ![add title label](./images/add-label-title.png)
 
-5. Modify the text for `Title` and adjust it's *position*, *text size* and *color*.
+5. Modify the properties of the `Title` node in the __Inspector__ panel as needed, such as `Position`, `Color`, `String`, `FontSize`, etc.
 
     ![modify title](./images/title-inspector.png)
 
-6. Adjust the position of the `PlayButton`. The layout of a simple __start menu__ is complete.
+6. Add as many Tips nodes as needed, then adjust the position and property values of each UI node, and a simple start menu is complete.
 
     ![modify title](./images/start-menu.png)
 
-7. Add game state logic, generally it can be divided into three states:
-    - **Init**: display the game menu and initialize some resources.
-    - **Playing**: hide the game menu, players can operate the game.
-    - **End**: end the game and display the ending menu.
+## Add game status logic
 
-    Use an enum type to represent these states.
+Add game state logic, generally it can be divided into three states:
 
-    ```ts
-    enum BlockType{
-        BT_NONE,
-        BT_STONE,
-    };
-
-    enum GameState{
-        GS_INIT,
-        GS_PLAYING,
-        GS_END,
-    };
-    ```
+1. **Init**: display the game menu and initialize some resources.
+2. **Playing**: hide the game menu, players can operate the game.
+3. **End**: end the game and display the ending menu.
 
-    Add a private variable that represents the current state to the `GameManager` script
+Each of these states can be represented using an `enum` type, adding the following to the `GameManager` script:
 
-    ```ts
-    private _curState: GameState = GameState.GS_INIT;
-    ```
+```ts
+enum BlockType{
+    BT_NONE,
+    BT_STONE,
+};
 
-    In-order not to let the user operate the character at the beginning, but to allow the user to operate the character while the game is in progress, we need to dynamically turn on and off the character's monitoring of mouse messages. This can be done with the following changes to `PlayerController`:
+enum GameState{
+    GS_INIT,
+    GS_PLAYING,
+    GS_END,
+};
+```
 
-    ```ts
-    start () {
-        // Your initialization goes here.
-        //systemEvent.on(SystemEvent.EventType.MOUSE_UP, this.onMouseUp, this);
-    }
+Then continue adding a private variable that represents the current state to the `GameManager` script.
 
-    setInputActive(active: boolean) {
-        if (active) {
-            systemEvent.on(SystemEvent.EventType.MOUSE_UP, this.onMouseUp, this);
-        } else {
-            systemEvent.off(SystemEvent.EventType.MOUSE_UP, this.onMouseUp, this);
-        }
+```ts
+private _curState: GameState = GameState.GS_INIT;
+```
+
+In order to allow the player to operate the character while the game is in progress rather than at the beginning of the game, we need to dynamically turn on/off the character's listening to mouse messages. Add the `setInputActive()` method to the `PlayerController` script.
```

Comment:
```suggestion
In order to allow the player to operate the character while the game is in progress rather than at the beginning of the game, it is necessary to dynamically turn on/off the character's listening to mouse messages. Add the `setInputActive()` method to the `PlayerController` script.
```